extensions/arc/src/ui/dashboards/postgres/postgresDashboard.ts

```diff
@@ -47,7 +49,7 @@ export class PostgresDashboard extends Dashboard {
 					propertiesPage.tab,
 					connectionStringsPage.tab,
 					computeAndStoragePage.tab,
-					parametersPage.tab
+					workerNodesParametersPage.tab
```

Comment:
What does this look like in the UI while we're hiding the per-role features?  The page should still say "Nodes Parameters" rather than "Worker Nodes Parameters" while it's still configuring both coordinator and workers.

Reply:
You are right, I added back in the original strings to the worker parameter page so that the UI looks the same way.